### PR TITLE
Lint diagnostic_tools package

### PR DESCRIPTION
**Pull request by hidmic**
Precisely what the title says. Purging technical debt.

### CODE REVIEW

File: diagnostics_tools/CMakeLists.txt [hunk 1]
```diff
@@ -114,6 +115,14 @@ install(DIRECTORY include/${PROJECT_NAME}/
...
 
+roslint_cpp()
+
+roslint_python()
+
+if(CATKIN_ENABLE_TESTING)
+  roslint_add_test()
+endif()
+
 ## Add gtest based cpp test target and link libraries
```
Comment (patapufate) [inline diagnostics_tools/CMakeLists.txt:126]: We should remove boilerplates and unused comments.

**Review by patapufate — CHANGES_REQUESTED** [review]

File: diagnostics_tools/CMakeLists.txt (hunk 1 above)
Comment (hidmic) [reply diagnostics_tools/CMakeLists.txt:126]: Good point.

Comment (hidmic) [reply diagnostics_tools/CMakeLists.txt]: See https://github.com/ChrisScianna/ROS-Underwater-RnD/pull/62/commits/9465ad229cddc6b8283c66632c2df4177247d350.

**Review by patapufate — APPROVED** [review]
LGTM!

**Comment by hidmic** [comment]
Alright, thanks for the review! Going in!

**Pull request merged**